src/components/dialog/Dialog.tsx

```diff
@@ -51,6 +51,14 @@ const Dialog = ({
 }: DialogProps) => {
   const bodyRef = useRef<HTMLElement>(null);
   const dragHandleRef = useRef<HTMLDivElement>(null);
+  const [showScrollShadow, setShowScrollShadow] = useState(true);
+  const [reachedScrollEnd, setReachedScrollEnd] = useState(false);
+  useEffect(() => {
+    const currentRef = bodyRef.current;
+    if (currentRef) {
+      setShowScrollShadow(currentRef.scrollHeight > currentRef.clientHeight);
+    }
+  }, [bodyRef, otherProps.active]);
```

Comment:
Does this implementation work when resizing the window?

Reply:
yes